CenterForOpenScience/osf.io
# [ENG-2397][Hotfix] RWE Registration Form update

## Purpose

Improves language for schema form.

## Changes

- simple textual change.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify language has changed in form.

## Documentation

🐞 fix, no docs

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2397
